cpp/include/cugraph/detail/graph_utils.cuh

```diff
@@ -78,19 +78,20 @@ rmm::device_uvector<edge_t> compute_major_degrees(
                                 [(detail::num_sparse_segments_per_vertex_partition + 2) * i +
                                  detail::num_sparse_segments_per_vertex_partition]
               : major_last;
-    thrust::transform(rmm::exec_policy(handle.get_stream())->on(handle.get_stream()),
+    rmm::exec_policy execution_policy = handle.get_thrust_policy();
```

Comment:
> Would this be better just to auto?

Probably. This would prevent the need for changes in case the `rmm::exec_policy` type is modified for something else in the future. I'll switch to using `auto`.

> Is there an advantage to storing the `execution_policy` as a variable over calling `handle.get_thrust_policy()` multiple times?

This allows to create the execution policy once and re-use it which should be slightly faster. Also, it makes the code a bit easier to read in my opinion.